fenix/app/src/main/java/org/mozilla/fenix/shopping/di/ReviewQualityCheckMiddlewareProvider.kt

```diff
@@ -36,4 +44,12 @@ object ReviewQualityCheckMiddlewareProvider {
         ),
```

Comment:
out of scope but could we inline ` reviewQualityCheckPreferences = ReviewQualityCheckPreferencesImpl(settings),` ?